app/views/paypal_accounts/_ask_paypal_permissions.haml

```diff
@@ -1,7 +1,8 @@
 .row
   = form.label :paypal_email, title
 .row
-  - community_country_code = LocalizationUtils.country_code_valid?(@current_community.country) ? @current_community.country.downcase : 'us' # default should not happen in production
+  - community_country_code = LocalizationUtils.valid_country_code(@current_community.country)
+
   - create = link_to(t("paypal_accounts.new.create_paypal_account"), "https://www.paypal.com/#{community_country_code}/signup")
   - upgrade = link_to(t("paypal_accounts.new.upgrade_paypal_account"), "https://www.paypal.com/#{community_country_code}/upgrade")
```

Comment:
Should these be changed to set in controller similarly as redirect link at the same time?